es-vue-base/src/lib-components/EsFileInput.vue

```diff
@@ -206,6 +206,10 @@ export default {
             this.pickedItems = [];
         },
         async verifyMimeType(file) {
+            // If a folder then trigger fileTypeError
```

Comment:
Do you mean "If not a folder"?

Reply:
Ahh it should be "if an empty folder" resolving it. Thank you for noticing this!